docs/reference/commandline/network_create.md

```diff
@@ -10,22 +10,27 @@ parent = "smn_cli"
 
 # network create
 
-    Usage:  docker network create [OPTIONS] NETWORK-NAME
-
-    Creates a new network with a name specified by the user
-
-    --aux-address=map[]      Auxiliary ipv4 or ipv6 addresses used by network driver
-    -d --driver=DRIVER       Driver to manage the Network bridge or overlay. The default is bridge.
-    --gateway=[]             ipv4 or ipv6 Gateway for the master subnet
-    --help                   Print usage
-    --internal               Restricts external access to the network
-    --ip-range=[]            Allocate container ip from a sub-range
-    --ipam-driver=default    IP Address Management Driver
-    --ipam-opt=map[]         Set custom IPAM driver specific options
-    --ipv6                   Enable IPv6 networking
-    --label=[]               Set metadata on a network
-    -o --opt=map[]           Set custom driver specific options
-    --subnet=[]              Subnet in CIDR format that represents a network segment
+```markdown
```

Comment:
heh, yes, not entirely accurate, but gave better highlighting 😇 
we should have a "plaintext" hint or something